backend/globaleaks/handlers/custodian/__init__.py

```diff
@@ -89,7 +90,7 @@ def update_identityaccessrequest(session, tid, user_id, identityaccessrequest_id
                                      models.IdentityAccessRequestCustodian.identityaccessrequest_id == models.IdentityAccessRequest.id,
                                      models.IdentityAccessRequestCustodian.custodian_id == user_id,
                                      models.InternalTip.id == models.IdentityAccessRequest.internaltip_id).one()
-
+    logging.debug(iarc)
```

Comment:
This file seems to be modified just to add a debugging line during development. Please revert it to not require overwelming security code review.